pkg/fanal/applier/applier.go

```diff
@@ -23,11 +23,15 @@ func NewApplier(c cache.LocalArtifactCache) Applier {
 
 func (a *applier) ApplyLayers(imageID string, layerKeys []string) (ftypes.ArtifactDetail, error) {
 	var layers []ftypes.BlobInfo
+	var layersInfo ftypes.Layers
```

Comment:
Actually, we forgot to rename `layers` to `blobs` when we renamed `LayerInfo` to `BlobInfo`. We can change it now so `layersInfo` can be renamed to `layers`.

```suggestion
	var blobs []ftypes.BlobInfo
	var layers ftypes.Layers
```

Reply:
I thought about this too, but decided not to do it because it might cause confusion for the `ApplyLayers` function:
https://github.com/aquasecurity/trivy/blob/75c193b8bfd962c4e0ac09b8b26f74ef2380f121/pkg/fanal/applier/applier.go#L38
I mean that we will use `ApplyLayers(blobs)` when there is `layers` variable.

We can rename ApplyLayers to ApplyBlobs, but that would require other renaming, so it's better to do it in another PR.

____

But if you sure in this renaming - tell me, i will update names.

Reply:
renamed in https://github.com/aquasecurity/trivy/pull/8394/commits/eb0780a4a38004de8faa886d31b8eb5e210590a6